Sources/Orbit/Components/SegmentedSwitch.swift

```diff
@@ -0,0 +1,214 @@
+import SwiftUI
+
+/// Showing two choices from which only one can be selected.
+///
+/// Currently SegmentedSwitch allows to have 2 segments.
+/// SegmentedSwitch can be in error state only in unselected state.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/interaction/segmentedswitch/)
+public struct SegmentedSwitch: View {
+
+    @Binding private var selectedIndex: Int?
+    let label: String
+    let message: Message?
+    let firstOption: String
+    let secondOption: String
+
+    let separatorWidth: CGFloat = BorderWidth.emphasis
+    let borderWidth: CGFloat = BorderWidth.emphasis
+
+    public var body: some View {
+        FieldWrapper(
+            label,
+            message: message
+        ) {
+            HStack(spacing: 0) {
+                segment(title: firstOption)
+                    .accessibility(.segmentedSwitchFirstOption)
```

Comment:
Here we can add overlay to the first segment to get rid of `fixedSize`

`.overlay(separator, alignment: .trailing)`

+

`separator...  { Color.cloudNormal.frame(width: borderWidth) }`